api/src/main/java/org/openmrs/module/muzima/handler/JsonRegistrationQueueDataHandler.java

```diff
@@ -13,19 +13,16 @@
  */
 package org.openmrs.module.muzima.handler;
 
+import com.jayway.jsonpath.Criteria;
+import com.jayway.jsonpath.Filter;
 import net.minidev.json.JSONArray;
+import net.minidev.json.JSONObject;
 import org.apache.commons.lang.StringUtils;
 import org.apache.commons.lang.time.DateUtils;
 import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
-import org.openmrs.Location;
-import org.openmrs.Patient;
-import org.openmrs.PatientIdentifier;
-import org.openmrs.PatientIdentifierType;
-import org.openmrs.PersonAddress;
-import org.openmrs.PersonAttribute;
-import org.openmrs.PersonAttributeType;
-import org.openmrs.PersonName;
+import org.javarosa.core.services.Logger;
+import org.openmrs.*;
```

Comment:
Could you avoid wildcard (*) imports? Your IDE should probably have a setting to disable automatically resorting to wildcard

Reply:
I am using Inteliji IDEA I believe there is a setting that can remove wildcard imports. I understand the confusion it can bring with Object mismatch and similar class names from different packages.